verification/io/dataplane_abstract.gobra

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +gobra
+
+package io
+
+type DataPlaneSpec adt {
+	DataPlaneSpec_{
+		linkTypes		dict[IO_ifs]IO_Link
+		neighborIAs		dict[IO_ifs]IO_as
+		localIA			IO_as
+		topology 		TopologySpec
+	}
+}
+
+//TopologySpec provides information about the entire network topology because
```

Comment:
Nitpick: in general, we should add a space between // and the comment text
```suggestion
// TopologySpec provides information about the entire network topology because
```